src/main/java/net/snowflake/ingest/streaming/internal/StreamingIngestResponseCode.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2021 Snowflake Computing Inc. All rights reserved.
+ */
+
+package net.snowflake.ingest.streaming.internal;
+
+public enum StreamingIngestResponseCode {
+  SUCCESS(0, "Success"),
+  ERR_INVALID_REQUEST(1, "The given request has an invalid format"),
+  ERR_CHANNEL_NAME_EXISTS(2, "The channel name already exists"),
+  ERR_INVALID_WRITE_MODE(3, "The supplied write mode is not valid"),
+  ERR_TABLE_DOES_NOT_EXIST_NOT_AUTHORIZED(
+      4, "The supplied table does not exist or is not authorized"),
+  ERR_CHANNEL_LIMIT_EXCEEDED_FOR_TABLE(5, "Channel limit exceeded for the given table"),
+  ERR_TABLE_OR_COLUMN_TYPE_NOT_SUPPORTED(
+      6,
+      "Snowpipe Streaming is not supported on the type of table resolved, or the table contains"
+          + " columns that are either AUTOINCREMENT, IDENTITY or has default value"),
+  ERR_ENQUEUE_TABLE_CHUNK_QUEUE_FULL(
+      7,
+      "Cannot enqueue chunks at this time due to a high number of chunks pending commit to the"
+          + " table"),
+  ERR_DATABASE_DOES_NOT_EXIST_OR_NOT_AUTHORIZED(
+      8, "The supplied database does not exist or is not authorized"),
+  ERR_SCHEMA_DOES_NOT_EXIST_OR_NOT_AUTHORIZED(
+      9, "The supplied schema does not exist or is not authorized"),
+  ERR_GENERAL_EXCEPTION_RETRY_REQUEST(
+      10, "Snowflake experienced a transient exception, please retry the request"),
+  ERR_MISSING_BLOBS_IN_REQUEST(11, "Missing the blobs array in the request"),
+  ERR_MISSING_CHUNKS_FOR_BLOB_IN_REQUEST(12, "Missing the chunks array in the request"),
+  ERR_MISSING_CHANNELS_FOR_CHUNK_FOR_BLOB_IN_REQUEST(
+      13, "Missing the channels array in the request"),
+  ERR_MISSING_BLOB_PATH(14, "The blob path was not supplied in the request"),
+  ERR_MISSING_DATABASE_IN_REQUEST(15, "A database was not supplied in the request"),
+  ERR_MISSING_SCHEMA_IN_REQUEST(16, "A schema was not supplied in the request"),
+  ERR_MISSING_TABLE_IN_REQUEST(17, "A table was not supplied in the request"),
+  ERR_MISSING_CHANNEL_NAME_IN_REQUEST(18, "A channel name was not supplied in the request"),
+  ERR_CHANNEL_NO_LONGER_EXISTS(19, "The requested channel no longer exists"),
```

Comment:
Let's change to `The requested channel no longer exists, most likely due to inactivity. Please re-open the channel`.